curr_scraper/apps/api_caller/models.py

```diff
@@ -1,3 +1,9 @@
 from django.db import models
 
-# Create your models here.
+
+class RateRecord(models.Model):
+    timestamp = models.DateTimeField('Date of record')
+    UAHrate = models.FloatField('UAH to USD')
+    GBPrate = models.FloatField('GBP to USD')
+    EURrate = models.FloatField('EUR to USD')
+    PLNrate = models.FloatField('PLN to USD')
```

Comment:
@Fazzer6132 No bueno, oh no bueno...

Please do not make such specific columns like UAHrate, GBPrate, EURrate, or PLNrate - you can always make a separate model that stores currencies and then make ForeignKey relations. Please normalize this entity.

Also, you are using USD as a base currency in all the cases, so please extend the models to be capable of storing all the pairs of currencies.